examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>After applying the quantum oracle, an H-gate is applied to all the qubits in the first register.</p><p> </p><p>This is because if there are three qubits in the first register, then the initial quantum state looks as follows after the first set of H-gates have acted on it:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>